frontend/src/app/modules/main/view-playlist/view-playlist.component.ts

```diff
@@ -0,0 +1,62 @@
+import { Playlist } from 'src/app/models/playlist/playlist';
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute, Params } from '@angular/router';
+import { Song } from 'src/app/models/song/song';
+import { PlaylistsService } from 'src/app/services/playlists/playlist.service';
+
+@Component({
+  selector: 'app-view-playlist',
+  templateUrl: './view-playlist.component.html',
+  styleUrls: ['./view-playlist.component.sass']
+})
+export class ViewPlaylistComponent implements OnInit {
+  public songs: Song[] = [];
+  public userId: number;
+  public totalCountSongs: number;
+  public hours: number;
+  public minutes: number;
+  public playlist: Playlist;
+  private _totalTimeSongs: number;
+  private _playlistId: number;
+
+  constructor(
+    private _activateRoute: ActivatedRoute,
+    private _playlistsService: PlaylistsService
+  ) { }
+
+  ngOnInit() {
+    this._activateRoute.params.subscribe((params: Params) => {
+      this._playlistId = params.id;
+      this.loadPlaylistSongs();
+      this.loadPlaylist();
+    });
+  }
+
+  nextSlide = () => {};
+
+  previousSlide = () => {};
+
+  play = () => {};
+
+  loadPlaylistSongs() {
+    this._playlistsService
+      .getPlaylistSongs(this._playlistId)
+      .subscribe((songs) => {
+        this.songs = songs;
+        this.totalCountSongs = songs.length;
+        this._totalTimeSongs = songs
```

Comment:
Violates B3.
https://github.com/BinaryStudioAcademy/quality-criteria/blob/production/source/javascript.md

Reply:
eslint - Class Property name `totalTimeSongs` must have one leading underscore(s).eslint@typescript-eslint/naming-convention
 this._totalTimeSongs - is private variable

Reply:
ok, will skip it for now